src/plugins/template/backend/ops/identity.cpp

```diff
@@ -0,0 +1,68 @@
+// Copyright (C) 2018-2024 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "openvino/reference/Identity.hpp"
+
+#include "Identity_shape_inference.hpp"
+#include "evaluate_node.hpp"
+
+template <ov::element::Type_t ET>
+inline bool evaluate(const std::shared_ptr<ov::op::v15::Identity>& op,
+                     ov::TensorVector& outputs,
+                     const ov::TensorVector& inputs) {
+    using T = typename ov::element_type_traits<ET>::value_type;
+
+    const std::vector<ov::PartialShape> input_shapes{op->get_input_shape(0)};
+    const auto total_size = get_shape_size(out_shape);
+    const auto total_size_in_bytes = total_size * inputs[0].get_dtype().get_element_size();
+
+    outputs[0].set_shape(input_shapes[0]);
+
+    ov::reference::Identity(static_cast<const char*>(inputs[0].data()), static_cast<char*>(outputs[0].data()), total_size_in_bytes);
+    return true;
+}
+
+template <>
+bool evaluate_node<ov::op::v15::Identity>(std::shared_ptr<ov::Node> node,
+                                          ov::TensorVector& outputs,
+                                          const ov::TensorVector& inputs) {
+    switch (node->get_input_element_type(0)) {
+    case ov::element::boolean:
+        return evaluate<ov::element::boolean>(ov::as_type_ptr<ov::op::v15::Identity>(node), outputs, inputs);
+    case ov::element::bf16:
+        return evaluate<ov::element::bf16>(ov::as_type_ptr<ov::op::v15::Identity>(node), outputs, inputs);
+    case ov::element::f16:
+        return evaluate<ov::element::f16>(ov::as_type_ptr<ov::op::v15::Identity>(node), outputs, inputs);
+    case ov::element::f64:
+        return evaluate<ov::element::f64>(ov::as_type_ptr<ov::op::v15::Identity>(node), outputs, inputs);
+    case ov::element::f32:
+        return evaluate<ov::element::f32>(ov::as_type_ptr<ov::op::v15::Identity>(node), outputs, inputs);
+    case ov::element::i4:
```

Comment:
Remove low precision they are usually not handled by reference implementation.

Reply:
Removed all <8bits precisions